### DISCUSSION TITLE
GPT_Researcher_Get_Stuck

### DISCUSSION THREAD

**Discussion by BioChemML** [assafelovic/gpt-researcher#382] [Q&A]
<img width="885" alt="Stuck" src="https://github.com/assafelovic/gpt-researcher/assets/100691244/fa263fa6-1719-4203-931d-c2403c4da8c3">
Why my gpt research get stuck here? Any idea how to solve this? Thank you!

**Comment by llmlocal** [comment]
Many things could cause this issue. There should be some error in the console log. In your browser select developer tools. Check the console and/or network tab. Rerun the prompt and if there is an error you should see it in one of those locations.
